rad/src/operators/string.rs

```diff
@@ -264,12 +280,62 @@ fn json_to_cbor(value: &json::JsonValue) -> Value {
     }
 }
 
+/// Replace thousands and decimals separators in a `RadonTypes` that is assumed to be a
+/// `RadonString`.
+///
+/// If the input value is not a `RadonString`, returns `RadError` because of `try_into`.
+pub fn replace_separators(
+    value: RadonTypes,
+    thousands_separator: serde_cbor::Value,
+    decimal_separator: serde_cbor::Value,
+) -> Result<RadonTypes, RadError> {
+    let rad_str_value: RadonString = value.try_into()?;
+
+    Ok(RadonTypes::from(RadonString::from(_replace_separators(
+        rad_str_value.value(),
+        thousands_separator,
+        decimal_separator,
+    ))))
+}
+
+/// Replace thousands and decimals separators in a `String`.
+pub fn _replace_separators(
```

Comment:
A public function starting with `_` looks weird, I would call it `replace_separators_string` or `replace_separators_inner` if it's private.